pkg/db/db.go

```diff
@@ -1069,6 +1070,29 @@ func (h *DBHandler) needsReleasesMigrations(ctx context.Context, transaction *sq
 
 }
 
+func (h *DBHandler) RunCustomMigrationReleasesTimestamp(ctx context.Context, getAllAppsFun GetAllAppsFun, fixReleasesTimestampFun FixReleasesTimestampFun) error {
+	span, ctx := tracer.StartSpanFromContext(ctx, "RunCustomMigrationReleases")
+	defer span.Finish()
+
+	return h.WithTransaction(ctx, false, func(ctx context.Context, transaction *sql.Tx) error {
```

Comment:
is it okay to run all of this within one transaction?
I think it might be too much, given how many release we have.